app/layouts/DashboardLayout/TopBar/NavBar.tsx

```diff
@@ -28,6 +29,20 @@ interface Section {
 }
 
 const sections: Section[] = [
+  ...(process.env.NODE_ENV === 'development' // TODO: REMOVE THIS!
+    ? [
+        {
+          Icon: ContactIcon,
+          label: 'ADD',
```

Comment:
Will this eventually be moved from the nav bar into the "Contacts" section, instead of having an Add and Edit option?